packages/devtools_app/lib/src/theme.dart

```diff
@@ -6,7 +6,10 @@ import 'package:flutter/material.dart';
 
 import 'common_widgets.dart';
 import 'config_specific/ide_theme/ide_theme.dart';
-import 'ui/theme.dart';
+
+const _contrastForegroundWhite = Color.fromARGB(255, 240, 240, 240);
+
+const contrastForegroundWhite = _contrastForegroundWhite;
```

Comment:
do we need both a private and a public const here?

Reply:
Did it because there was use of same value Color.fromARGB(255, 240, 240, 240) inside of the extension and contrastForegroundWhite was getter name in extension so outer constant name couldn't be used.  Now there is one definition for contrastForegroundWhite @ _contrastForegroundWhite.